docs/managing-airbyte/connector-updates.md

```diff
@@ -0,0 +1,64 @@
+# Managing Connector Updates in Airbyte
+
+While maintaining an Airbyte instance, you'll need to manage connector updates. These are essential for improved functionality and reliability over time. Our team and community contributors are dedicated to maintaining and improving the functionality of connectors. Reasons for updates can be broadly categorized into bug fixes, new features, and changes that impact the user experience or the functionality of the connector.
+
+This guide helps you understand the types of updates you may see, their impact on your Airbyte environment, and the actions you may need to take in response to certain types of updates.
+
+## Understanding Connector Versions 
+To manage connection updates effectively, it's important to understand versioning and how to interpret the changelog entries. 
+
+Every connector in Airbyte's catalog follows semantic versioning (semver)
+Major.Minor.Patch (e.g., 1.2.5)
+A connector reaching version 1.0 is considered mature and comes with semver guarantees. 
+
+**Minor updates (1.2.x):** These might introduce new features like streams or properties, but they are designed to be fully backward compatible with your existing setup.<br/>
+**Patch updates (1.x.x):** These typically contain bug fixes and small improvements that won't affect your existing configurations.<br/>
+**Major updates (x.0.0):** These are significant changes that may require you to adjust your configurations. We'll discuss this in more detail below.<br/>
+
+:::info
+When a connector version is 1.0 or higher, this means it's mature and now has semver guarantees.
+:::
+
+Each connector's changelog details its update history. You can find it in the [connector catalog](../integrations/) at the end of each individual connector's entry.
+
+<!-- maybe insert Arcade clip navigating to changelog and toggling it open for revew -->
+
+## How Airbyte Handles Connector Updates
+
+### Airbyte Cloud
+**Minor and Patch Updates:** These are applied automatically and immediately to your instance. You don't need to take any action.
+
+**Major Updates:** These are opt-in. You'll receive notifications about major updates and have a deadline to decide whether to apply them. If you choose not to opt-in by the deadline, any syncs using the affected connector will be paused to prevent compatibility issues.
```

Comment:
```suggestion
**Major Updates:** These are opt-in. You'll receive notifications about major updates and have a deadline to apply them. If you choose not to opt-in by the deadline, any syncs using the affected connector will be paused to prevent compatibility issues.
```